src/ILVerify/src/ILImporter.StackValue.cs

```diff
@@ -194,7 +194,7 @@ bool IsAssignable(TypeDesc src, TypeDesc dst, bool allowSizeEquivalence = false)
             }
 
             if (dst.IsInterface || dst.IsArray)
-                throw new NotImplementedException();
+                throw new NotImplementedException($"{nameof(IsAssignable)} is only partially implemented.");
 
             // TODO: Other cases - variance, etc.
```

Comment:
@jkotas Should we update the TODO to guide people towards reusing `src\Common\src\TypeSystem\Common\CastingHelper.cs`? I assume we don't want this to grow into a separate copy of the casting logic. Casting is hard.